### ISSUE TITLE
[A11y] 1.4.1 Use of Colour - Links Lack Additional Styling

### ISSUE THREAD

**Issue by dalonsoa** [research-software-directory/RSD-as-a-service#1760]
See https://github.com/ImperialCollegeLondon/RSD-as-a-service/issues/47 for details

**Comment by dmijatovic** [comment]
Can you handle this one please?

**Comment by dalonsoa** [comment]
This issue was actually fixed by https://github.com/research-software-directory/RSD-as-a-service/pull/1796, so closing.
